examples/examples_go_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build go all
+
+package main
+
+import (
+	"github.com/pulumi/pulumi-eks/examples/utils"
+	"github.com/pulumi/pulumi/pkg/v2/testing/integration"
+	"path/filepath"
+	"testing"
+)
+
+func TestAccClusterGo(t *testing.T) {
+	test := getGoBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: filepath.Join(getCwd(t), "cluster-go"),
+			ExtraRuntimeValidation: func(t *testing.T, info integration.RuntimeValidationStackInfo) {
+				utils.RunEKSSmokeTest(t,
+					info.Deployment.Resources,
+					info.Outputs["kubeconfig1"],
+					info.Outputs["kubeconfig2"],
+				)
+			},
+		})
+
+	integration.ProgramTest(t, &test)
+}
+
+func getGoBaseOptions(t *testing.T) integration.ProgramTestOptions {
+	region := getEnvRegion(t)
+	base := getBaseOptions(t)
+	goBase := base.With(integration.ProgramTestOptions{
+		Config: map[string]string{
+			"aws:region": region,
+		},
+		Dependencies: []string{
+			"Pulumi.Eks",
```

Comment:
This is specific to DotnEt

Reply:
We'll need to fix this up, but we can do that when we hook up the test to CI, separately.

Reply:
I made the change but I expect to need to fiddle here as part of CI work.